pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -172,7 +172,7 @@ func (a *Bootstrap) addBootstrapFiles(dependencies asset.Parents) {
 	)
 	a.Config.Storage.Files = append(
 		a.Config.Storage.Files,
-		ignition.FileFromString("/opt/tectonic/report-progress.sh", 0555, content.ReportShFileContents),
+		ignition.FileFromString("/usr/local/bin/tectonic-report-progress.sh", 0555, content.ReportShFileContents),
```

Comment:
This isn't "Tectonic" any more, is it?  Do we expect `report-progress.sh` to conflict with something that the host supplies?  I'd expect `/usr/local/bin` to be empty on a fresh RHCOS, and even if it isn't, the bootstrap node is something that we use briefly and throw away; would it cause problems even if we got very unlucky and there was an existing `/usr/local/bin/report-progress.sh`?

Reply:
> This isn't "Tectonic" any more, is it?

No but let's not scope in a major rename into this?  As far as the UX goes the systemd units are a lot more visible.

> Do we expect report-progress.sh to conflict with something that the host supplies? 

No, but why not namespace it anyways?  It's just general best practice.

Reply:
> > This isn't "Tectonic" any more, is it?
>
> No but let's not scope in a major rename into this? As far as the UX goes the systemd units are a lot more visible.

I'm fine letting existing names ride, but would rather not add additional technical debt in the "Tectonic" direction ;).

> > Do we expect report-progress.sh to conflict with something that the host supplies?
>
> No, but why not namespace it anyways? It's just general best practice.

I like namespacing, but `/usr/local/bin` is *already* sufficient namespacing for me, because it's [for commands installed locally by the system administrator][1], that system administrator is the installer, and the bootstrap node should be gone within 30 minutes of creation ;).  The existing `report-progress.sh` and `bootkube.sh` already describe what those scripts do; we're unlikely to have other scripts installed locally by the system administrator that do those same tasks.  So why add extra namespacing?

Still, this is a pretty minor point.  If other maintainers want to roll ahead with additional namespacing, `tectonic-` or otherwise, I'll survive ;).

[1]: http://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s09.html

Reply:
To be clear I'm fine if you (or someone else) wants to force-push a different naming to this PR.

Reply:
I notice you didn't suggest a new naming prefix though...since I'm already in here it's tempting to call it `tectonicshift-bootstrap.sh` with the new name *I* think the project should have... :wink:

Reply:
> To be clear I'm fine if you (or someone else) wants to force-push a different naming to this PR.

Done with 1b8bc5b -> be338f7.  This is going to be pretty embarrassing for me if we end up having a naming collision ;).